src/Model/Tag.php

```diff
@@ -282,6 +306,114 @@ private static function getTypeForHash(string $hash)
 		} else {
 			return self::UNKNOWN;
 		}
+	}
+
+	/**
+	 * Retrieves the terms from the provided type(s) associated with the provided item ID.
+	 *
+	 * @param int       $item_id
+	 * @param int|array $type
+	 * @return array
+	 * @throws \Exception
+	 */
+	public static function ArrayFromURIId(int $uri_id, array $type = [self::HASHTAG, self::MENTION, self::IMPLICIT_MENTION, self::EXCLUSIVE_MENTION])
```

Comment:
Please use `camelCase` for function names.
```suggestion
	public static function getByURIId(int $uri_id, array $type = [self::HASHTAG, self::MENTION, self::IMPLICIT_MENTION, self::EXCLUSIVE_MENTION])
```